src/pages/ExamsPage/messages.js

```diff
@@ -230,6 +230,12 @@ const messages = defineMessages({
     defaultMessage: 'Submission Reason: ',
     description: 'Submission reason label for exam attempt data in the review/reset modal',
   },
+
+  ReviewDashboardOpenLTITool: {
+    id: 'ExternalReviewDashboard.open_lti_tool',
+    defaultMessage: 'View resource in a new window',
```

Comment:
Is there something else that I should set the default message to? Something like "View instructor tool in a new window"? Not sure of the correct language to use.

Reply:
either sound just fine to me.